lib/queuePopulator/LogReader.js

```diff
@@ -330,19 +320,47 @@ class LogReader {
             }
             return key;
         }
-        if (entry.key === undefined || entry.value === undefined) {
-            return;
+
+        function _executeFilter(ext, entry, cb) {
+            if (typeof ext.filterAsync === 'function') {
```

Comment:
Also this change does not seem to be in the proper commit, as you modified the changes made in the first commit

Reply:
I made this change after our discussion. Do you mean the other PR?

Reply:
Yes, the other PR or commit that is included in the other PR